lib/process-services-cloud/src/lib/task/task-filters/services/task-filter-cloud.service.ts

```diff
@@ -379,4 +380,29 @@ export class TaskFilterCloudService extends BaseCloudService {
     refreshFilter(filterKey: string): void {
         this.filterKeyToBeRefreshedSource.next(filterKey);
     }
+
+    private handleBackwardCompatibility(appName: string, key: string, filters: TaskFilterCloudModel[]): Observable<TaskFilterCloudModel[]> {
```

Comment:
This method will filre at the start of the app, when we retrieve the filters from local storage or when we first populate them.
If we don't find the new keys in the retrieved filters, but we find their old counterparts we populate the new keys and leave the counterparts as they are.